mintapi/api.py

```diff
@@ -1301,7 +1305,7 @@ def get_credit_report(self, limit=2, details=True, exclude_inquiries=False):
 
         self._load_mint_credit_url()
 
-        credit_report["reports"] = self._get_credit_reports(limit, credit_header)
+        # credit_report["reports"] = self._get_credit_reports(limit, credit_header)
```

Comment:
Is this removal intentional?

Reply:
Nope. Got caught in my last test. Thanks for that catch.